public/index.html

```diff
@@ -21,37 +21,12 @@
   -->
 
   <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=G-CP65J0T1G0"></script>
-    <script>
-    window.dataLayer = window.dataLayer || [];
-    function gtag(){dataLayer.push(arguments);}
-    gtag('js', new Date());
 
-    gtag('config', 'G-CP65J0T1G0');
-  </script>
 
   <script src="https://unpkg.com/web-streams-polyfill@2.0.4/dist/ponyfill.es6.js"></script>
 
-  <script type="text/javascript">
-    (function(e, t, r, n, a) {
-      var c = [];
-      e[a] = function() {
-        c.push(arguments);
-      };
-      e[a].queue = c;
-      var s = t.createElement(r);
-      s.async = 1;
-      s.src = n;
-      var u = t.getElementsByTagName(r)[0];
-      u.parentNode.insertBefore(s, u);
-    })(window, document, 'script', '//cdn.sematext.com/rum.js', 'strum');
-  </script>
-  <script type="text/javascript">
-    strum('config', {
-      token: '155abb23-9f7e-4bf0-bebd-86ee4043ed74',
-      receiverUrl: 'https://rum-receiver.eu.sematext.com',
-    });
-  </script>
+
```

Comment:
guessing these enters are left over accidentally, lets remove them to keep the file clean

Reply:
gracias